WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -637,7 +637,7 @@ private extension DefaultProductFormTableViewModel {
 
         // No price warning row
         static let noPriceWarningTitle =
-            NSLocalizedString("Add price to your variations to make them visible on your store",
+            NSLocalizedString("Add a price to your variation to make them visible on your store",
```

Comment:
```suggestion
            NSLocalizedString("Add a price to your variation to make it visible on your store",
```

Just a small tweak to consistently remove pluralization 😊

Reply:
Thanks! 😅